src/app/dashboard-widgets/add-codebase-widget/add-codebase-widget.component.html

```diff
@@ -57,10 +54,7 @@ <h2 class="card-pf-title">
       <div class="f8-blank-slate-card" *ngIf="codebases.length === 0">
         <h3>This Space has no Codebases</h3>
         <div class="f8-blank-slate-main-action">
-          <button id="spacehome-my-codebases-create-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
-        </div>
-        <div class="f8-blank-slate-main-action">
-          <button id="spacehome-my-codebases-import-button" class="btn btn-primary btn-lg" [routerLink]="[{ outlets: { action: 'add-codebase' } }]">Import Codebase</button>
```

Comment:
Similarly, why was this removed from the old widget?

Reply:
I think this is touched in the long discussion of https://github.com/openshiftio/openshift.io/issues/3021

https://github.com/openshiftio/openshift.io/issues/3100#issuecomment-380884315
suggests import codebase functionality can be removed.

On 3021: Alex says:

"Additionally, if we want a fix in place for summit but haven't fully hashed out the details of url patterns, would it be acceptable to make a patch that removes the add-codebases action (or replace it's functionality with displaying the forge-wizard), and have the add-work-item "overlay" be it's own page?"

Joshua replies

"I think it would be. That would at least not break user flow."


So I think these are intentional removals.

Reply:
I still think if you can't fix it fast, then removing it for now is best.

@catrobson @qodfathr do you agree?

Reply:
+1 that Add Codebase should go to Launcher Experience (if that flag is enabled for that user on the button) or to the old Forge wizard (if the feature flag is not enabled for that user on the button).  If we can't go to the old forge wizard, removing the button from the old widget would be acceptable to me over keeping the old 'create codebase' fly-out panel.  @dlabrecq does this make sense to you - I want to make sure I'm understanding this correctly.

Reply:
There are two issues here; the "import codebase" button and the "add work item" icon. 

It's still not clear as to why the "import codebases" button was removed because it still works for me without any changes. If the team wants to remove the "import codebase" button, I'm fine with that. I understand that it's not needed with the new app-launcher, but believe that's outside the scope of this PR.

Regarding the "add work item" icon, I think that should be removed. Even with these fixes, the user cannot create a work item because two APIs no longer exists. I was able to work around one of them, but not the second missing AP. These APIs are no longer in the swagger docs.

Reply:
I think the issue with the "import codebase" is that if you use it and then route to pipelines or deployments, your URL looks something like:

`https://openshift.io/jkang@redhat.com/arstneio/create/(pipelines//action:add-codebase)`

and the header menu then highlights the Codebases tab even when you're on Pipelines or Deployments. However, as far as I can tell, you're right, everything else works.

Reply:
That functionality looks okay to me. Although, it seems we didn't have these routing issues until the two router-outlets were added to app-component.html?

Reply:
Huh; I really can't say re. the router outlets being added and these issues being seen. That's another thing to investigate I guess.